### PR TITLE
Add crypto-square exercise

### PR DESCRIPTION
**Pull request by jimmbraddock**

### CODE REVIEW

**Comment by dalexj** [comment]
I think I've got a much simpler version of the example for you:

```elixir
defmodule CryptoSquare do
  @doc """
  Encode string square methods
  ## Examples

    iex> CryptoSquare.encode("abcd")
    "ac bd"
  """
  @spec encode(String.t) :: String.t
  def encode(""), do: ""
  def encode(str) do
    normalized = normalize_string(str)
    section_length = normalized |> byte_size |> :math.sqrt |> Float.ceil |> trunc

    normalized
    |> String.graphemes
    |> Enum.chunk(section_length, section_length, List.duplicate("", section_length))
    |> List.zip
    |> Enum.map(&Tuple.to_list/1)
    |> Enum.join(" ")
  end

  defp normalize_string(str) do
    str
    |> String.downcase
    |> String.replace(~r/[^a-z0-9]/, "")
  end
end
```

**Comment by jimmbraddock** [comment]
Yes, your solution is simpler. Should I replace my solution?

File: exercises/crypto-square/crypto_square_test.exs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+defmodule CryptoSquareTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "empty string" do
+    assert CryptoSquare.encode("") == ""
+  end
+
+  @tag :pending
+  test "perfect_square" do
```
Comment (parkerl) [inline exercises/crypto-square/crypto_square_test.exs]: remove the underscore here please.

**Comment by jimmbraddock** [comment]
I eliminated the remarks and replaced the example on @dalexj variant.

File: exercises/crypto-square/crypto_square_test.exs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    assert CryptoSquare.encode("") == ""
+  end
+
+  @tag :pending
+  test "perfect square" do
+    assert CryptoSquare.encode("abcd") == "ac bd"
+  end
+
+  @tag :pending
+  test "upcase string" do
```
Comment (parkerl) [inline exercises/crypto-square/crypto_square_test.exs]: Uppercase

**Comment by jimmbraddock** [comment]
Fixed.

**Comment by parkerl** [comment]
Thanks 🎉

**Pull request merged**